GsaGHTests/1_BaseParameters/2_Geometry/GsaElement2dTest.cs

```diff
@@ -386,5 +400,52 @@ private GsaElement2d CreateSampleElement2dWithQuad4Type() {
 
       return elem;
     }
+
+    private GsaElement2d CreateLoadPanel() {
+      var pts = new Point3dList {
+       new Point3d(0, 0, 0),
+       new Point3d(2, 0, 0),
+       new Point3d(2, 2, 0),
+       new Point3d(0, 2, 0),
+     };
+
+      pts.Add(pts[0]);
+      var pol = new Polyline(pts);
+      var elem = new GsaElement2d(pol.ToPolylineCurve());
+
+      int initialElementId = 14;
+      int sectionId = 3;
+      var groups = new List<int>();
+      var dummy = new List<bool>();
+      var names = new List<string>();
+      var off = new List<GsaOffset>();
+      elem.Prop2ds = new List<GsaProperty2d>();
+      for (int i = 0; i < elem.ApiElements.Count; i++) {
+        elem.Ids[i] = initialElementId++;
+        groups.Add(22);
+        dummy.Add(true);
+        names.Add("Shahin");
+        elem.Prop2ds.Add(new GsaProperty2d(sectionId) {
+          ApiProp2d = new Prop2D(),
+        });
+      }
+
+      elem.ApiElements.SetMembers(groups);
+      elem.ApiElements.SetMembers(dummy);
+      elem.ApiElements.SetMembers(names);
+      return elem;
```

Comment:
The next Part, could we wrap it into a separate method? Are the "22" for group, dummy "true" significant for what we are testing or can we use the same? It seems like they are not.

I am noticing that sometimes we use "off" sometimes we don't could we add those as effects.

```
var element2d = GetElement2dFrom(mesh);
setDefaults(element2d);
applyOff(element2d);
```